5_databases/week1/homework.sql

```diff
@@ -0,0 +1,61 @@
+-- Active: 1699100471406@@0.0.0.0@3306@hyf_lesson1
+
+-- Task 1 Find out how many tasks are in the task table
+SELECT COUNT (title)
```

Comment:
No space between COUNT and (title). And just use * instead of title because you dont have to specify a certain field here